tests/common.py

```diff
@@ -172,3 +173,16 @@ def __init__(self, instances):
     def get_all_instances(self, filters=None):
         self.filters = filters
         return [Reservation({"instances": [i for i in self.instances]})]
+
+
+class pytest_regex:
```

Comment:
there really isn't anything here wrt to pytest, doing a straight value comparison with self.assertRegex seems simpler then introducing new test machinery

Reply:
Fair enough, updated test.